lib/headlines/security_headers/x_frame_options.rb

```diff
@@ -15,7 +15,7 @@ def score_by_value
       end
 
       def score_by_allow_from
-        whitelisted_domain? ? 1 : 0
+        whitelisted_domain? ? 1 : -1
```

Comment:
@codelion Ok, if so I will revert it (will wait @CalebFenton answer). Thank you.